src/content/en/tools/_redirects.yaml

```diff
@@ -24,6 +24,12 @@ redirects:
 - from: /web/tools/chrome-devtools/console/console-reference
   to: /web/tools/chrome-devtools/console/api
 
+- from: /web/tools/lighthouse/v3/scoring
+  to: https://web.dev/performance-scoring/
+  
+- from: /web/tools/lighthouse/scoring
```

Comment:
This redirect will not work because the original content is still there. DevSite will only redirect if the path has been removed.